unit_testing/test_straightening.py

```diff
@@ -11,16 +11,16 @@
 sys.path.append(os.path.join(__sct_dir__, 'scripts'))
 from spinalcordtoolbox.straightening import SpinalCordStraightener
 import sct_utils as sct
-
+from spinalcordtoolbox.utils import sct_test_path
 
 VERBOSE = 0  # Set to 2 to save images, 0 otherwise
 
 
 # noinspection 801,PyShadowingNames
 def test_straighten():
     """Test straightening with default params"""
-    fname_t2 = os.path.join(sct.__sct_dir__, 'sct_testing_data/t2/t2.nii.gz')  # sct_download_data -d sct_testing_data
-    fname_t2_seg = os.path.join(sct.__sct_dir__, 'sct_testing_data/t2/t2_seg-manual.nii.gz')
+    fname_t2 = sct_test_path('t2/t2.nii.gz')  # sct_download_data -d sct_testing_data
+    fname_t2_seg = sct_test_path('t2/t2_seg-manual.nii.gz')
```

Comment:
since we're at it (for our windows friends)
```suggestion
    fname_t2 = sct_test_path('t2', 't2.nii.gz')  # sct_download_data -d sct_testing_data
    fname_t2_seg = sct_test_path('t2', 't2_seg-manual.nii.gz')
```

Reply:
not strictly necessary, with WSL windows uses forward slashes as well, but good form anyway